scripts/scil_tractogram_project_map_to_streamlines.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Projects metrics extracted from a map onto the endpoints of streamlines.
```

Comment:
This is great, much clearer than having many options in scil_project_streamlines_to_map.

You should remove option --in_metrics in that other script. We can now do it in two (clearer) steps:

1) Your new script to project metric to dpp
2) Old script to project back to map.

Reply:
All other comments addressed. 

I started to remove --in_metrics from project_streamlines_to_map but then realized I may have to worry about legacy usage of this script.  Is it fine just to remove it and assume people that used this option will have to use an old version of scilpy?